apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -101,6 +101,11 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
     }
   }, [eventType.customInputs]);
 
+  const eventNamePlaceholder = t("meeting_with_user")
+    .replace("{Event type title}", eventNameObject.eventType)
```

Comment:
This is a duplicate of the code above at https://github.com/calcom/cal.com/pull/7247/files#diff-e6807e04030a7e08ae8814936a8b27eba0efbb12277eba305e2bda7e25a42396R88. 
Let's extract it out and reuse everywhere.

Reply:
pushed those changes